### PR TITLE
Resolve conflicts

### PR DESCRIPTION
**Pull request by KanchanaJayasinghe**

### CODE REVIEW

File: frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.admin-table {
+    padding: 0 2vw;
+  }
+  
+  .custom-table {
+    align-content: center;
+  }
```
Comment (deshankoswatte) [inline frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.css]: New line at the end of file.

File: frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.js [hunk 2]
```diff
@@ -1,37 +1,53 @@
 import React from "react";
-import ReactDataGrid from "react-data-grid";
+// import Loading from "../../../../../utility/loading/loading";
+// import axios from "axios";
+// import ViewRequest from "./view_request/view_request";
+// import Comment from "./comment/comment";
```
Comment (deshankoswatte) [inline frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.js]: Why commented imports?

File: frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.css (hunk 1 above)
Comment (KanchanaJayasinghe) [reply frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.css]: Resolved.

File: frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.js (hunk 2 above)
Comment (KanchanaJayasinghe) [reply frontend/web/src/components/home/body/dashboard/admin_dash/grid/admin_grid.js]: Removed commented lines.

**Review by deshankoswatte — APPROVED** [review]

**Pull request merged**